pontoon/base/models.py

```diff
@@ -2330,6 +2358,58 @@ def cleaned_key(self):
     def __str__(self):
         return self.string
 
+    def stats_states_map(self, locale):
+        translations = list(
+            self.translation_set
+                .filter(locale=locale)
+                .prefetch_related('errors', 'warnings')
+        )
+
+        approved_strings_count = len([
+            t for t in translations
+            if t.approved
+        ])
+        fuzzy_strings_count = len([
+            t for t in translations
+            if t.fuzzy
+        ])
+        unrevieved_count = len([
+            t for t in translations
+            if not (t.approved or t.fuzzy or t.rejected)
+        ])
```

Comment:
Move this block under `errors_count` and `warnings_count` please, so that we keep the same order as in the output diff and `adjust_stats` params.